partcad-cli/src/partcad_cli/click/commands/list/parts.py

```
@@ -23,20 +23,27 @@
         part_kinds = 0
 
         if used_by is not None:
-            logging.info("Instantiating %s..." % used_by)
-            ctx.get_assembly(used_by)
+            pc_logging.info("Instantiating %s..." % used_by)
+            try:
+                target = ctx.get_assembly(used_by)
+                if not target:
+                    pc_logging.error(f"Assembly {used_by} is not found")
+                    return
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)